felix_requirements.txt

```diff
@@ -1,4 +1,4 @@
 gevent
 greenlet
 netaddr
-python-etcd==0.3.3-calico-1
+python-etcd==0.3.3-calico-2
```

Comment:
I wonder if there's a case for doing that for gevent too, for Debian compatibility. We don't have Debian as a target platform for any of our packaged install methods; but we might want to help Debianites trying out Calico from source.

Although - does that mean we'd also have to add cython here?